modules/caddyhttp/reverseproxy/command.go

```diff
@@ -64,6 +64,7 @@ default, all incoming headers are passed through unmodified.)
 			fs.Bool("insecure", false, "Disable TLS verification (WARNING: DISABLES SECURITY BY NOT VERIFYING SSL CERTIFICATES!)")
 			fs.Bool("internal-certs", false, "Use internal CA for issuing certs")
 			fs.Bool("debug", false, "Enable verbose debug logs")
+			fs.Bool("disable-redirects", false, "Disable redirect HTTP->HTTPS")
```

Comment:
Might be me nitpicking, but I would rephrase it ever so slightly :woman_shrugging: 

```suggestion
			fs.Bool("disable-redirects", false, "Disable HTTP->HTTPS redirects")
```

Reply:
@IndeedNotJames I took care of it